class-prdd-lite-woocommerce.php

```diff
@@ -376,6 +419,192 @@ public function prdd_lite_review_text() {
 			<tr>
 			<?php
 		}
+
+		/**
+		 * This function change the meta_keys to make compliant with pro plugin.
+		 *
+		 * @since 2.3.0
+		 */
+		public function prdd_lite_update_data_for_pro() {
+			global $prdd_lite_update_checker;
+			?>
+			<div class="wrap about-wrap">
+				<?php /* translators: %s: version number */ ?>
+				<h2><?php printf( esc_html__( 'Welcome to Product Delivery Date for WooCommerce - Lite v%s', 'woocommerce-prdd-lite' ), esc_attr( $prdd_lite_update_checker ) ); ?></h2>
+				<div>
+				<p><?php esc_html_e( 'We have noticed that you have updated the version of the Product Delivery Date for WooCommerce - Lite on your store. Thus, before activating the plugin, we request you to upgrade the database.', 'woocommerce-prdd-lite' ); ?></p>
+				<p><?php esc_html_e( 'You can choose if you want to continue or not? Click on Yes to continue from the options below:', 'woocommerce-prdd-lite' ); ?></p>
+				</div>
+				<input type="button" id="prdd-update-yes" class="button button-primary" value="Yes"  />
+				<input type="button" id="prdd-update-no" class="button button-primary" value="No"  />
```

Comment:
Remove the 'No' option & it's associated code. Change the 'Yes' label to 'Update database'.